src/src/rx_main.cpp

```diff
@@ -1180,7 +1160,12 @@ bool ICACHE_RAM_ATTR ProcessRFPacket(SX12xxDriverCommon::rx_status const status)
         return false;
     }
 
-    PFDloop.extEvent(beginProcessing + PACKET_TO_TOCK_SLACK);
+    // The extEvent defines where TOCK timer ISR is to be synced to, i.e. where the packet period begins.
+    // For rates where the TOA is longer than half the packet period schedule the TOCK for rougly 1x TOA before
+    // the TX's end of the period so telemetry is received by the TX in the correct period. For all others,
+    // schedule TOCK to be PACKET_TO_TOCK_SLACK (us) after RX packet reception.
+    int32_t slack = std::max(ExpressLRS_currAirRate_Modparams->interval - 2 * ExpressLRS_currAirRate_RFperfParams->TOA, (int32_t)PACKET_TO_TOCK_SLACK);
```

Comment:
Something to check is how well this plays with the transmitters Rxnb timeout, which is set to 1 interval.

With this change and at close range the transmitter will have finished receiving tlm right at the end of the timeout, which is ok because the timeout is cancelled on preamble/sync.  Add delay due to ToF and you have artificially caused a tlm loss due to the preamble not being received before the timeout.

I know I know... its F500 and who TF is flying at any kind of range.  But this is the type of thing to bite us later.  So it may be worthwhile looking at the transmitters timeout interval and having it cancel a period of TOA before the next packet Txnb.